native/mod_proxy_cluster/mod_proxy_cluster.c

```diff
@@ -226,45 +303,48 @@ static apr_status_t create_worker(proxy_server_conf *conf, proxy_balancer *balan
         return APR_EGENERAL;
     }
     if (uri.port && uri.port == ap_proxy_port_of_scheme(uri.scheme)) {
-        uri.port = 0;
+        url = apr_pstrcat(pool, node->mess.Type, "://", normalize_hostname(pool,node->mess.Host), NULL);
     }
-    ptr = apr_uri_unparse(pool, &uri, APR_URI_UNP_REVEALPASSWORD);
 
-    worker = ap_proxy_get_worker(pool, balancer, conf, ptr);
+    worker = ap_proxy_get_worker(pool, balancer, conf, url);
     if (worker == NULL) {
 
         /* creates it note the ap_proxy_get_worker and ap_proxy_define_worker aren't symetrical, and this leaks via the conf->pool */ 
-        const char *err = ap_proxy_define_worker(conf->pool, &worker, balancer, conf, url, 0);
+        const char *err;
+        ap_log_error(APLOG_MARK, APLOG_DEBUG, 0, server,
+                     "Created: worker for %s Will create %d!!!", url, node->mess.id);
+        err = ap_proxy_define_worker(conf->pool, &worker, balancer, conf, url, 0);
         if (err) {
             ap_log_error(APLOG_MARK, APLOG_NOTICE|APLOG_NOERRNO, 0, server,
                          "Created: worker for %s failed: %s", url, err);
             return APR_EGENERAL;
         }
 
         worker->context = (proxy_cluster_helper *) apr_pcalloc(conf->pool,  sizeof(proxy_cluster_helper));
-        if (!worker->context)
+        if (!worker->context) {
             return APR_EGENERAL;
+        }
         helper = (proxy_cluster_helper *) worker->context;
         helper->count_active = 0;
         helper->shared = worker->s;
+        helper->isinnodes = 0;
         ap_log_error(APLOG_MARK, APLOG_DEBUG, 0, server,
                      "Created: worker for %s", url);
     } else {
+        ap_log_error(APLOG_MARK, APLOG_DEBUG, 0, server,
+                     "Created: worker for %s Already exist!!!", url);
         if (!worker->context) {
-            /* That is BalancerMember */
+            /* That is BalancerMember, we dropped support of it */
             ap_log_error(APLOG_MARK, APLOG_DEBUG, 0, server,
                          "Created: reusing BalancerMember worker for %s", url);
```

Comment:
Maybe add a mention of dropping support into the log too?

Reply:
there is a JIRA https://issues.redhat.com/browse/MODCLUSTER-771 the down stream was fixed some time ago.